flyway_test.go

```diff
@@ -0,0 +1,221 @@
+package flyway_test
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/CyberOwlTeam/flyway"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/go-connections/nat"
+	"github.com/google/uuid"
+	_ "github.com/lib/pq"
+	"github.com/stretchr/testify/require"
+
+	"github.com/testcontainers/testcontainers-go"
+	tcpostgres "github.com/testcontainers/testcontainers-go/modules/postgres"
+	tcnetwork "github.com/testcontainers/testcontainers-go/network"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	defaultPostgresDbVersion  = "16.3"
+	defaultPostgresPort       = "5432"
+	defaultPostgresSrvName    = "pgdb"
+	defaultPostgresDbName     = "test_db"
+	defaultPostgresDbUsername = "postgres"
+	defaultPostgresDbPassword = "postgres"
+)
+
+type intPostgresContainer struct {
+	*tcpostgres.PostgresContainer
+}
+
+func (c *intPostgresContainer) getNetworkUrl() string {
+	return fmt.Sprintf("jdbc:postgresql://%s:%s/%s?sslmode=disable", defaultPostgresSrvName, defaultPostgresPort, defaultPostgresDbName)
+}
+
+func (c *intPostgresContainer) getExternalUrl(ctx context.Context) (string, error) {
+	url, err := c.ConnectionString(ctx)
+	if err != nil {
+		return "", err
+	}
+	return fmt.Sprintf("%ssslmode=disable", url), nil // disable ssl
```

Comment:
Is this correct? There is no any separator between `%s` and `sslmode`

Reply:
it is ... the url ends with a slash - that was one of the things Manuel changed, to use ConnectionString from the testcontainer